code/modules/mob/living/simple_animal/hostile/gorilla/gorilla.dm

```diff
@@ -36,6 +36,10 @@
 	var/list/carriable_cache
 	/// A lazylist of all crates we are carrying
 	var/list/atom/movable/crates_in_hand
+	var/dismember_chance = 10
+	var/stamina_damage = 20
+	///5 hits to stam crit
+	var/throw_onhit = 50
```

Comment:
Please explain what these vars do in an autodoc comment

Reply:
Stoniest is a newer contributor, heres how autodoc should look like `/// ` and then the comment
```suggestion
        /// The chance to dismember when punching
	var/dismember_chance = 10
	/// How much stamina damage will be delt if the person attacked is not thrown
	var/stamina_damage = 20
	/// The chance to throw when punching
	var/throw_onhit = 50
```

Reply:
All good, I just wanted to make sure there was no unanswered questions for them